core/controllers/story_viewer_test.py

```diff
@@ -579,7 +579,7 @@ def test_mark_topic_as_partially_learnt_and_story_as_completed(self):
             )
 
         self.assertEqual(len(
-            learner_progress_services.get_all_partially_learnt_topic_ids(
+            learner_progress_services.get_all_learnt_topic_ids(
```

Comment:
The test description no longer matches what you're checking here.

Reply:
Done.